service-monitor/pom.xml

```diff
@@ -16,20 +16,6 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.scala-lang</groupId>
-            <artifactId>scala-library</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.reactivex</groupId>
```

Comment:
Remove all rxjava dependencies from parent pom.xml

Reply:
I failed to find any dependencies in parent pom.xml that had to be removed from these changes, except the one you commented on below. Is there any particular dependency you're referring to?